wordpress-mobile/WordPress-Android
# Issue/4727 notifs detail view horizontal navigation

Fixes #4727 

This PR implements 2 ways to horizontally navigate through Notifications detail:
- using swipe left/right (by means of a ViewPager control)
- touching the left/right arrows now available on the action bar

To test:
1. log in to the WP Android app with your account
2. go to the Notifications tab (rightmost tab)
3. you'll be presented with the notifications list.
4. tap on any item -  you'll be presented with the item's detail screen
5. swipe left-right to navigate. 
6. Note that you'll be able to navigate left-right items as they are listed top-bottom in the notifications list correspondingly. That means if you tapped on the first item in the list, you won't be able to swipe left on that item (as you can't go any further up while already looking at the top of the list). Same thing for the last item in the list.
7. Also, if you tap on any of the filters (All, Comments, unread, etc.)  you'd be able to navigate the list horizontally accordingly - that is, items are filtered there as well.  

See video here https://cloudup.com/cBZOd5sFriD

cc @daniloercoli @jleandroperez @kwonye
